### PR TITLE
Additional Fields and Entity Type for Lighting Fixtures

### PR DESCRIPTION
**Pull request by bsaxpi**
New Abstract "Color Temperature Control"
New Entity Type "LT_DC_CTC_OM"

### CODE REVIEW

File: ontology/yaml/resources/LIGHTING/entity_types/LT.yaml [hunk 1]
```diff
@@ -72,3 +72,12 @@ LT_DM_CHSLM_OM:
   - DM
   - CHSLM
   - OM
+
+LT_DC_CTC_OM:
```
Comment (cstirdivant) [inline ontology/yaml/resources/LIGHTING/entity_types/LT.yaml:76]: Please add guid

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 2]
```diff
@@ -3014,6 +3014,12 @@ literals:
 - lighting_color_hsl_hue_angle_setpoint:
     fixed_min: 0.0
     fixed_max: 360.0
+- lighting_color_temperature_setpoint:
+    flexible_min: 0.0
+    flexible_max: 10000.0
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:3019]: A max of 10,000 K is equivalent to 9726 degrees Celsius, which seems quite high

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/LIGHTING/entity_types/LT.yaml (hunk 1 above)
Comment (bsaxpi) [reply ontology/yaml/resources/LIGHTING/entity_types/LT.yaml:76]: ...added

File: ontology/yaml/resources/fields/telemetry_fields.yaml (hunk 2 above)
Comment (bsaxpi) [reply ontology/yaml/resources/fields/telemetry_fields.yaml]: Indeed. Changed to max. 7k. Should be sufficient.

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**